### ISSUE TITLE
Disable TabComplete not working properly.

### ISSUE THREAD

**Issue by CasieBarie** [PedroJM96/SuperLobbyDeluxe#24]
Hello,
Recently I noticed the 'Disable TabComplete' hides all the commands while you have some commands whitelisted.
While looking through the source code I found the problem!

```java
e.getCommands().clear();
e.getCommands().addAll(filter); 
```
You are using this to first clear the commands and then add back the whitelisted ones. But Bukkit doesn't allow you to add back commands. You can only remove them.

To fix this you have to add all the commands in a list except the whitelisted ones. Then you just remove them.
### Example:
```java
ArrayList<String> toRemove = new ArrayList<>();
List<String> whitelist = plugin.config.getStringList("disable-tab-complete.whitelist");
for(String command : e.getCommands()) {
    if(!whitelist.contains("/" + command)) {
        toRemove.add(command);
    }
}
e.getCommands().removeAll(toRemove);
```

I hope this helps to fix the problem.
Have a nice day!

**Comment by CasieBarie** [comment]
This is also fix for #20

**Comment by PedroJM96** [comment]
Fixed in the next version, thanks for the code
